src/__tests__/RegularExpression.test.js

```diff
@@ -44,6 +42,56 @@ describe('RegularExpression', () => {
       });
     });
 
+    describe('does not match (w/options)', () => {
```

Comment:
Maybe add a test when null/undefined or even another data type is passed if you want to be that verbose? 🤷‍♂️